sbom.cdx.json

```diff
@@ -0,0 +1,43 @@
+{
```

Comment:
Since there's no obligation for the file to be at the toplevel, I think we shouldn't clutter the toplevel directory. We don't really have a clearly applicable directory, but how about `scripts`, alongside `*requirements.txt`?

Reply:
Yes, scripts works for me, thanks!